src/plots/gl2d/scene2d.js

```diff
@@ -286,10 +292,11 @@ proto.updateFx = function(options) {
 
     fullLayout.dragmode = options.dragmode;
     fullLayout.hovermode = options.hovermode;
-};
 
-var relayoutCallback = function(scene) {
+    this.graphDiv._fullLayout = fullLayout;
```

Comment:
would it be an option to not use `this.fullLayout` at all, but only to use `this.graphDiv._fullLayout`?